python/llm/setup.py

```diff
@@ -39,7 +39,7 @@
 from setuptools import setup
 
 long_description = '''
-    IPEX LLM
+    IPEX-LLM is a PyTorch library for running LLM on Intel CPU and GPU (e.g., local PC with iGPU, discrete GPU such as Arc, Flex and Max) with very low latency
```

Comment:
please also add this comment: latency**[1](https://github.com/intel-analytics/ipex-llm#user-content-fn-1-3fb86d9a951feffdcd8d94b9c6712529)**.